Zend/tests/named_params/named_paramater_alias_attribute_001.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+NamedParameterAlias Attribute
+--FILE--
+<?php
+
+function test(#[NamedParameterAlias("bar")] $foo) {
```

Comment:
There should be more tests and checks in the implementation, e.g.
1. Should invalid identifiers such as `NamedParameterAlias("some-name")` throw or be silently forbidden for invalid variadic arguments?
2. Should php warn/throw if using an alias for a parameter name that was already used? (`test(#[NamedParameterAlias("bar")] $foo = null, $bar = null)`)
3. Handling supporting more than one alias?